src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,70 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @OneToOne
+    @JoinColumn(name = "cinema_hall_id")
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
+
+    public MovieSession() {
```

Comment:
Don't create a default constructor when it is not needed. If you are not using the default constructor or it's not required by a framework, it's better to avoid adding unnecessary code.